diesel/src/macros/as_changeset.rs

```diff
@@ -259,8 +261,10 @@ macro_rules! AsChangeset_construct_changeset_ty {
         fields = [{
             $(field_name: $field_name:ident,)*
             column_name: $column_name:ident,
-            field_ty: Option<$field_ty:ty>,
+            field_ty: $ignore:ty,
             field_kind: option,
+            inner_field_ty: $field_ty:ty,
+            $($rest:tt)*
```

Comment:
FYI you commented above this

> FIXME: This can be cleaned up significantly when https://github.com/rust-lang/rust/issues/27245 is stable

Well, <https://github.com/rust-lang/rust/issues/27245> is stable now!